redacted-compiler-plugin/build.gradle

```diff
@@ -1,10 +1,11 @@
 apply plugin: 'org.jetbrains.kotlin.jvm'
 apply plugin: 'org.jetbrains.kotlin.kapt'
+apply plugin: 'maven-publish'
 
 compileKotlin {
   kotlinOptions {
     jvmTarget = "1.8"
-    freeCompilerArgs = ['-Xskip-runtime-version-check', 'true']
+    //freeCompilerArgs = ['-Xskip-runtime-version-check', 'true']
```

Comment:
When upgrading to Kotlin 1.3.61, kapt would fail with autoservice.  Disabling this resolved that, the error I encountered was cryptic and not worth debugging.  After this change, clicking the Run button of the sample project stopped working in Intellij.